fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/ValidatorContext.java

```diff
@@ -36,6 +38,19 @@
      */
     public ValidationResult result;
 
+    public boolean validateWith(Function<FluentValidator, FluentValidator> configuration) {
+        ComplexResult localResult = configuration.apply(FluentValidator.checkAll())
```

Comment:
I think I prefer the other solution that we came up with now. (using composition)